docs/sources/examples/_index.md

```diff
@@ -0,0 +1,13 @@
+---
+title: Grafana Loki Examples
+menuTitle: Examples
+description: Configuration reference for the parameters used to configure Grafana Loki.
```

Comment:
update description. "Examples to help configure Grafana Loki"?

Reply:
Done :)